### ISSUE TITLE
new resource-abuse list initial update problem

### ISSUE THREAD

**Issue by joey04** [gorhill/uBlock#3070]
I don't use any 3rd party lists in my main browser. With all the recent list changes I thought to test it today from a clean uBO database by importing my settings.

So I then let the browser idle for a while with the logger open, and this is what I see:

![untitled](https://user-images.githubusercontent.com/14959062/31038744-7ca65648-a546-11e7-92b2-1e527fca54c1.png)

Resource-abuse was also enabled in the Dashboard (which I assume is correct since it's a new default list entry, whereas none of the other new optional lists are selected). So then clicking the update button does indeed update it. But it appears to not be able to auto-update on its own initially.

edit: It will eventually auto-update through the normal means.

**Comment by gorhill** [comment]
There is no issue. You do not have `resource-abuse.txt` in your package, where uBO first try to fetch it. So it falls back to fetch it from remote location. By design.

**Comment by joey04** [comment]
> So it falls back to fetch it from remote location

No it didn't. I had to manually click the button for it to be fetched (it was empty 0 of 0 otherwise). Hence filing this issue.

**Comment by joey04** [comment]
I picked up the latest 1.14.11rc10 changes today, including assets, and did the same new database then import settings process. Now it's no longer selecting resource-abouse because its present in the extension .xpi package, as gorhill stated above. So hopefully that puts this little issue to bed for me and my particular use case.

I also edited my initial post to mention that in my original scenario it would eventually auto-update properly by normal means of asset fetching. (I still think it should do it immediately after importing settings, but this is a rather minor issue given that the vast majority of users probably just have the default settings.)
